simpl/config.py

```diff
@@ -408,16 +444,35 @@ def parse_ini(self, paths=None):
                       If not supplied, uses the ini_paths value supplied on
                       initialization.
         """
+        namespace = namespace or self.prog
         results = {}
         config = ConfigParser.SafeConfigParser()
-        config.read(paths or self._ini_paths)
+
+        if os.path.isfile(self.default_ini) and (
+                self.default_ini not in self._ini_paths):
+            self._ini_paths.append(self.default_ini)
+
+        config.read(paths or reversed(self._ini_paths))
         for option in self._options:
             ini_section = option.kwargs.get('ini_section')
+            value = None
             if ini_section:
                 try:
                     value = config.get(ini_section, option.name)
                     results[option.dest] = option.type(value)
-                except ConfigParser.NoSectionError:
+                except ConfigParser.NoSectionError as err:
+                    print('Error parsing ini file: %r -- Continuing.'
```

Comment:
I don't know that this class should be assuming it can own outputting to stdout. It should probably LOG and let the app or deployer decide how that output should be routed. Especially if I have `--quiet` as an argument.